src/common/components/InfoModal/InfoModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+    Alert,
+    AlertDescription,
+    AlertIcon,
+    AlertTitle,
+    Button,
+    Modal,
+    ModalContent,
+    ModalOverlay
+} from "@chakra-ui/react";
+
+interface Props {
+    isError?: boolean;
+    message: string;
+    title: string;
+    onClose: () => void;
+}
+
+export const InfoModal = (props: Props) => {
+    const {title, message, onClose, isError} = props;
+    return (
+        <Modal onClose={onClose} isOpen={true} isCentered>
+            <ModalOverlay/>
+            <ModalContent borderRadius="8px" borderColor={isError ? 'red.200' : 'green.200'} p="0.5rem"
+                          bgColor={isError ? 'red.200' : 'green.200'}>
```

Comment:
Generalnie Chakra pozwala na takie rzeczy, ale jest to mało eleganckie pchać tyle propsów. Lepiej wziąć to i upakować w osobne komponenty, w których dopiero są faktycznie dzieci - tu np. InfoModalu. Tak, żeby iny komponent przysłonił te propsy typowo Chakrowe z rodzica.